apps/teams-test-app/e2e-test-data/stageView-self.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "StageView-Self",
+  "platforms": "Web",
+  "version": ">2.0.0-beta.3",
+  "checkIsSupported": {
+    "expectedOutput": "StageView Self Actions are not supported",
+    "version": ">2.0.0"
+  },
+  "testCases": [
+    {
+      "title": "close API Call - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_stageView.self.close",
+      "inputValue": {
```

```suggestion
      "hostSdkVersion": {
        "web": ">3.1.0"
      }
      "inputValue": {
```

Something like this will make sure your e2e test only runs when used against a host running a version of the host-sdk that has your new functionality.